api/main.go

```diff
@@ -83,6 +84,11 @@ func main() {
 func hostApi(socketHarOutputChannel chan<- *tap.OutputChannelItem) {
 	app := fiber.New()
 
+	app.Use(cors.New(cors.Config{
+		AllowOrigins: "*",
+		AllowMethods: "*",
+		AllowHeaders: "*",
+	}))
```

Comment:
I don't know enough about CORS to tell if this is somehow a security risk.
I guess that because Mizu server does not authenticate, then we are not making anything worse by allowing CORS?

Reply:
yes, it is mainly to be able to access mizu when starting the site locally and want to work against server that running in another port